File: benchmarks/poly/jsonserialize/jsonserialize.php
<?php
$_t0 = hrtime(true);
$parts = [];
for ($i = 0; $i < 400000; $i++) {
    $parts[] = '{"id":' . $i . ',"name":"user' . $i . '","score":' . ($i % 100) . '}';
}
$out = implode("\n", $parts);
echo strlen($out) . "\n";
echo count($parts) . "\n";
fwrite(STDERR, "bench_ns=" . (hrtime(true) - $_t0) . "\n");

File: benchmarks/poly/jsonserialize/jsonserialize.lua
local _t0 = os.clock()
local parts = {}
for i = 0, 400000 - 1 do
    parts[#parts + 1] = '{"id":' .. i .. ',"name":"user' .. i .. '","score":' .. (i % 100) .. '}'
end
local out = table.concat(parts, "\n")
print(#out)
print(#parts)
io.stderr:write(string.format("bench_ns=%.0f\n", (os.clock() - _t0) * 1e9))
